lib/adyen/rest/request.rb

```diff
@@ -29,17 +29,19 @@ module REST
     # @see Adyen::REST::Client
     # @see Adyen::REST::Response
     class Request
-      attr_reader :prefix, :form_data, :required_attributes
+      attr_reader :prefix, :form_data, :required_attributes, :resource
       attr_accessor :response_class, :response_options
 
       def initialize(action, attributes, options = {})
-        @prefix = options[:prefix]
-        @form_data = generate_form_data(action, attributes)
+        raise Adyen::REST::RequiredParameterMissing, "actions is required!" if action.nil?
```

Comment:
Backward compatibility is already broken by the removal of the `:prefix` option. However, using an `action:` parameter makes no difference since people can still pass `action: nil`.

However, let me point out again that this check is not necessary to prevent users of this class from passing a nil action.